libcxx/src/filesystem/posix_compat.h

```diff
@@ -300,15 +296,15 @@ inline int statvfs(const wchar_t* p, StatVFS* buf) {
       break;
     path parent = dir.parent_path();
     if (parent == dir) {
-      errno = ENOENT;
+      SetLastError(ERROR_PATH_NOT_FOUND);
```

Comment:
Why do you `SetLastError` here but everywhere else you only `return -1`? Same question a few times below.